packages/office-addin-manifest/src/export.ts

```diff
@@ -31,7 +38,36 @@ async function createZip(manifestPath: string): Promise<AdmZip> {
     throw new Error(`The file '${manifestPath}' does not exist`);
   }
 
-  const manifest: devPreview.DevPreviewSchema = await ManifestUtil.loadFromPath(manifestPath);
+  const manifest: DevPreviewSchema = readJsonFileSync(manifestPath);
```

Comment:
So does the `ManifestUtil.loadFromPath` not work with the locally created schema then?

Reply:
I believe the return type was different. There's no real value to using their function as I needed to use the readJsonFileAsync() function for both JSON files. Seems simplest to use the same function for both.